src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,43 @@
 import React from 'react';
+import './PostInfo.scss';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+import { Post } from '../../types/Post';
+
+type Props = {
+  post: Post,
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    body,
+    user,
+    comments = [],
```

Comment:
it's not clear why did you set an empty array here by default
`comments` property in post entity can't be undefined